Add Météo France icons + device_class

Quentame opened the pull request:
## Breaking Change:

None

## Description:

Add Météo France icons

<img width="402" alt="Capture d’écran 2019-09-05 à 01 45 19" src="https://user-images.githubusercontent.com/14821482/64300868-e10c0100-cf7e-11e9-960c-e0d552a34c75.png">


## Checklist:
  - [x] The code change is tested and works locally.
  - [x] Local tests pass with `tox`. **Your PR cannot be merged unless tests pass**
  - [x] There is no commented out code in this PR.
  - [x] I have followed the [development checklist][dev-checklist]

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html

Quentame committed:
Add Météo France icons

pvizeli commented:
You could also add the correct device_class for the sensor if for that type is available? https://developers.home-assistant.io/docs/en/entity_sensor.html#available-device-classes

Quentame committed:
Add Météo France device_class (temperature) + use constants

Quentame commented:
@pvizeli : updated but the only available device class is temperature so I did it but don't know if it's really worth it. Also I refactored all the const "variables" from the __init__.py file to a const.py file and use a `dict` instead of an `array` for the `SENSOR_TYPES`.

pvizeli commented on homeassistant/components/meteo_france/sensor.py:
```diff
@@ -79,7 +88,9 @@ def __init__(self, condition, client, alert_watcher):
-        return "{} {}".format(self._data["name"], SENSOR_TYPES[self._condition][0])
+        return "{} {}".format(
+            self._data["name"], SENSOR_TYPES[self._condition][SENSOR_TYPE_NAME]
+        )
```
use new `f"{...} {...}` format

Quentame committed:
Fix weather alert info log

Quentame committed:
Use new f"{...} {...}" format for sensor name

Quentame commented:
Can someone look at my tiny little cutie PR ? 😊
@pvizeli @MartinHjelmare
